### ISSUE TITLE
Poor accuracy of network v1

### ISSUE THREAD

**Issue by tinybighuman** [niemtec/Effects-of-Image-Augmentation-on-CNNs#1]
Project-Turing/binary-cancer-classifier-v1/classifier-network-v1.py

The network version 1.0 has achieved reasonable prediction accuracy after 100 epochs, however when it is exposed to training data said accuracy drops considerably.

![model-loss](https://user-images.githubusercontent.com/10573413/46470193-b42ef700-c7cd-11e8-8bec-8210b832b8c9.png)
![model-accuracy](https://user-images.githubusercontent.com/10573413/46470194-b42ef700-c7cd-11e8-9814-4b577601e6d9.png)
